tests/CMakeLists.txt

```diff
@@ -29,7 +29,8 @@ if (ENABLE_GTEST)
         serac_boundary_cond.cpp
         serac_mesh.cpp
         mfem_ex9p_blockilu.cpp
-        serac_newmark_test.cpp)
+	serac_newmark_test.cpp
```

Comment:
indenting.

Reply:
Sorry. I really need to change my default emacs settings.